package.json

```
@@ -712,6 +709,11 @@
           "type": "boolean",
           "default": false,
           "description": "Output the action that VSCode should perform as requested by the server in JSON format."
+        },
+        "objectscript.ignoreInstallServerManager": {
+          "type": "boolean",
+          "default": false,
+          "markdownDescription": "Do not offer to install `#intersystems-community.servermanager#` extension"
```

The syntax you used is only for hyperlinking to other settings.
```suggestion
          "markdownDescription": "Do not offer to install the [intersystems-community.servermanager](https://marketplace.visualstudio.com/items?itemName=intersystems-community.servermanager) helper extension."
```